skipruntime-ts/examples/departures.ts

```diff
@@ -48,7 +42,11 @@ class DeparturesResource implements Resource {
       resettlement: get("resettlement", "NOR,USA"),
     };
 
-    return context.useExternalResource("http", "departures", params);
+    return context.useExternalResource(
+      "externalDeparturesAPI",
+      "departures",
+      params
+    );
```

Comment:
Not specific to this diff, but I wonder if `useExternalResource` would be clearer if we defined an object type for its arguments so that calls would look something like:
```suggestion
    return context.useExternalResource({
      supplier: "externalDeparturesAPI",
      resource: "departures",
      params: params
    });
```

Reply:
Good point: https://github.com/SkipLabs/skip/pull/408